### PR TITLE
chore: bump cardano-node to 1.35.7

### PR DESCRIPTION
**Pull request by iccicci**
# Context

A new version of `cardano-node` was released.

# Proposed Solution

Bump the image version our docker compose.

# Important Changes Introduced

Added to the doc some info about possible `cardano-node` OOM related problems.

### CODE REVIEW

File: packages/cardano-services/README.md [hunk 1]
```diff
@@ -144,6 +144,15 @@ RABBITMQ_SRV_SERVICE_NAME=some-domain-for-rabbitmq \
 The _Docker images_ produced by the SDK and the _docker compose infrastructures_ (_mainnet_, _preprod_ and _local-network_) it includes are ready to be used in
 production environment.
 
+To use _mainnet_, `cardano-node` my require a huge amount of RAM; this can lead to be killed due to OOM. The solution may be platform dependant:
```
Comment (rhyslbw) [inline packages/cardano-services/README.md:147]:
```suggestion
Running `cardano-node` on _mainnet_ requires a minimum of 24GB RAM.
```

Keep it simple 😃 
https://github.com/input-output-hk/cardano-node/releases/tag/1.35.7

**Review by rhyslbw — CHANGES_REQUESTED** [review]

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Typo in the commit message too
Screenshot 2023-07-25 at 17 53 16

File: packages/cardano-services/README.md (hunk 1 above)
Comment (iccicci) [reply packages/cardano-services/README.md]: 31a7c161 should solve this

File: packages/cardano-services/README.md [hunk 2]
```diff
@@ -144,6 +144,15 @@ RABBITMQ_SRV_SERVICE_NAME=some-domain-for-rabbitmq \
 The _Docker images_ produced by the SDK and the _docker compose infrastructures_ (_mainnet_, _preprod_ and _local-network_) it includes are ready to be used in
 production environment.
 
+Running [cardano-node](https://github.com/input-output-hk/cardano-node/releases/tag/1.35.7) on _mainnet_ requires a minimum of 24GB RAM.
```
Comment (rhyslbw) [inline packages/cardano-services/README.md:147]:
```suggestion
Running [cardano-node] on _mainnet_ requires a minimum of 24GB RAM.
```

Best to not add maintenance overhead here. The suggestion will add the link sourced [here](https://github.com/input-output-hk/cardano-js-sdk/pull/827/files#diff-8190db84f584e67953bdc740053321b6a4d4ede419843a7bd47f90bab2729fc4R208)

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: packages/cardano-services/README.md (hunk 2 above)
Comment (iccicci) [reply packages/cardano-services/README.md]: Nice, thank you!
5c4d690 should solve this

**Review by rhyslbw — APPROVED** [review]
Thanks @iccicci

**Pull request closed without merging**